js/modules/k6/html/gen/gen_elements.go

```diff
@@ -38,192 +49,192 @@ var funcDefs = []struct {
 	Elem, Method, Attr, TemplateType string
```

Comment:
Make TemplateType a `type TemplateType string` and define constants with that type instead.